compose.py

```diff
@@ -91,7 +92,8 @@ def add_requested_backends(ddir, dockerfile_name, backends):
     for backend in backends:
         df += '''COPY --chown=1000:1000 --from=full /opt/tritonserver/backends/{} /opt/tritonserver/backends/{}    
 '''.format(backend, backend)
-    df += '''
+    if len(backends) > 0:
```

Comment:
len is always +ve (>= 0), so len != 0 should suffice

Reply:
Is this just a style preference?